pyproject.toml

```diff
@@ -183,3 +183,8 @@ known-third-party = [
     "requests", "scipy", "setuptools", "shapely", "skimage", "sklearn", "streamlit",
     "torch", "torchvision", "tqdm", "typer"
 ]
+
+[tool.ragbits.core.default_llm_factories]
```

Comment:
Why do we want this in the library's main pyproject?

Reply:
Without this tests are failing, some of them require this default functions + I think its good to have an example how to set it up for the users

Reply:
Which tests? This seems like a problem with the tests - it means that the test will give different results depending the directory from which the command is run, which is not good.

Reply:
Ok. This should test should fail, because it alerts us to a bug in the implementation.

You changed the previous implementation to rely on default image LLM provider to always be set, which is not a true assumption for user projects. We actually want tests that makes sure that the library doesn't fail for projects without default LLM providers specified in their pyproject.toml files.